tembo-cli/src/cmd/login.rs

```diff
@@ -171,3 +211,91 @@ pub fn update_access_token(
 
     Ok(())
 }
+
+fn update_context(org_id: &str, profile_name: &str) -> Result<()> {
+    let context_file_path = tembo_context_file_path();
+    let mut contents = fs::read_to_string(&context_file_path)?;
+    let mut data: Context = toml::from_str(&contents)?;
+
+    if let Some(env) = data.environment.iter_mut().find(|p| p.set == Some(true)) {
+        env.set = Some(false);
+    }
+
+    match data.environment.iter().any(|p| p.name == profile_name) {
+        true => {
+            error(&format!("Context Environment already exists. Either try creating a new name or set it to that."));
```

Comment:
@sjmiller609 Could you confirm if this error message is good when user tries to create a duplicate profile? Thank you :)